imaging/tools/segmentation.js

```diff
@@ -191,6 +190,21 @@ export function setActiveLabelmap(labelId, elementId) {
   setters.activeLabelmapIndex(element, labelId);
 }
 
+// check if there is an active labelmap for target element
+/**
+ * Get active labelmap for target element
+ * @param {String} elementId
+ * @returns {Object} The active labelmap object that contains the buffer
+ */
+export function getActiveLabelmapBuffer(elementId) {
+  if (!elementId) {
+    console.error(
+      "getActivelabelmapBuffer now needs elementId as param and returns the buffer parent object"
+    );
+  }
```

Comment:
In order to be consistent with other functions that uses elementId in larvitar, use the isElement function inside

Reply:
`import { isElement } from "../imageUtils";`

`let element = isElement(elementId)
    ? elementId
    : document.getElementById(elementId);
  if (!element) {
    console.error("invalid html element: " + elementId);
    return;
  }`